src/test/webapp/WEB-INF/node1-node-deferred.xml

```diff
@@ -28,7 +28,7 @@
         </init-param>
         <init-param>
             <param-name>sticky-session</param-name>
-            <param-value>false</param-value>
```

Comment:
I think we should have tests running for the value `false` as well. Can you please add it?